### ISSUE TITLE
[a11y] Product search results should also expose scores to accessibility users.

### ISSUE THREAD

**Issue by SpacyRicochet** [openfoodfacts/openfoodfacts-ios#932]
# Description
Users with accessibility needs should be able to get an accurate summary of the product in their search results. Currently, VoiceOver users only hear the product name, brand name and quantity; but the various scores visible to other users are not read out loud.

E.g. VoiceOver speaks:

> Spaghetti n. 5, Bardille, 500 grams

# Acceptance criteria

VoiceOver should also include the various visible scores in its description. E.g;

> Spaghetti n. 5, Bardille, 500 grams, Nutri-Score 5, Processed culinary ingredients, some impact to the environment.

### Tasks
- [ ] Give the NutriScore an accessibility summary.
- [ ] Give the NovaGroup rating an accessibility summary.
- [ ] Give the Eco-Score an accessibility summary.
- [ ] Add the score's summaries to the cell's accessibility label.
- [ ] The navigation through the search results should not be impacted.
